remote-query/remote-query-client/src/main/java/org/infinispan/query/remote/client/impl/Externalizers.java

```diff
@@ -27,9 +27,9 @@ public void writeObject(ObjectOutput output, QueryRequest queryRequest) throws I
          output.writeUTF(queryRequest.getQueryString());
          output.writeLong(queryRequest.getStartOffset() != null ? queryRequest.getStartOffset() : -1);
          output.writeInt(queryRequest.getMaxResults() != null ? queryRequest.getMaxResults() : -1);
-         output.writeInt(queryRequest.hitCountAccuracy() != null ? queryRequest.hitCountAccuracy() : -1);
          output.writeObject(queryRequest.getNamedParameters());
          output.writeBoolean(queryRequest.isLocal());
+         output.writeInt(queryRequest.hitCountAccuracy() != null ? queryRequest.hitCountAccuracy() : -1);
```

Comment:
aren't these changes breaking backward compatibility? Protostream can handle out-of-order fields, but externalizers don't.

Reply:
IMO, this change should be in the main branch either. I'm guessing this affects clients using jboss-marshalling. cc @tristantarrant

Reply:
This part of the change is about the @tristantarrant, I would let him reply. I 'm more interested in the other change (the one to Protobuf clients).